src/handlers/discord-events/guild-create.ts

```diff
@@ -0,0 +1,65 @@
+import { ActionRowBuilder, AuditLogEvent, ButtonBuilder, ButtonStyle, Client, EmbedBuilder } from 'discord.js';
+import { isNil, noop } from 'lodash';
+
+export default (client: Client) => {
+  client.on('guildCreate', async (guild) => {
+    const auditLogCheck = (await guild.fetchAuditLogs({ limit: 1, type: AuditLogEvent.BotAdd})).entries.first()?.executorId || guild.ownerId;
```

Comment:
Errors are always to be handled correctly.

https://github.com/xp-bot/xp-bot/blob/7f4a87a0fcc7477d4e5211f72a37276a21d0bf6a/CONTRIBUTING.md?plain=1#L31

Even tho there i have yet to implement a comprehensive logging solution, please at least mark this with `console.error(...`